app/controllers/drafts/ViewAllDraftMovementsController.scala

```diff
@@ -0,0 +1,115 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.drafts
+
+import cats.data.EitherT
+import config.{AppConfig, ErrorHandler}
+import connectors.emcsTfe.GetDraftMovementsConnector
+import connectors.referenceData.GetExciseProductCodesConnector
+import controllers.helpers.BetaChecks
+import controllers.predicates.{AuthAction, AuthActionHelper, BetaAllowListAction, DataRetrievalAction}
+import forms.ViewAllDraftMovementsFormProvider
+import models.draftMovements.GetDraftMovementsSearchOptions.DEFAULT_MAX_ROWS
+import models.draftMovements.{DraftMovementSortingSelectOption, GetDraftMovementsSearchOptions}
+import models.requests.DataRequest
+import models.response.emcsTfe.draftMovement.GetDraftMovementsResponse
+import models.response.{ErrorResponse, NotFoundError}
+import play.api.data.Form
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
+import viewmodels.draftMovements.DraftMovementsPaginationHelper
+import viewmodels.helpers.SelectItemHelper
+import views.html.viewAllDrafts.ViewAllDraftMovementsView
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class ViewAllDraftMovementsController @Inject()(mcc: MessagesControllerComponents,
+                                                getDraftMovementsConnector: GetDraftMovementsConnector,
+                                                view: ViewAllDraftMovementsView,
+                                                getExciseProductCodesConnector: GetExciseProductCodesConnector,
+                                                errorHandler: ErrorHandler,
+                                                val auth: AuthAction,
+                                                val getData: DataRetrievalAction,
+                                                val betaAllowList: BetaAllowListAction,
+                                                formProvider: ViewAllDraftMovementsFormProvider,
+                                                paginationHelper: DraftMovementsPaginationHelper
+                                          )(implicit val executionContext: ExecutionContext, appConfig: AppConfig)
+  extends FrontendController(mcc)
+    with AuthActionHelper with I18nSupport with BetaChecks {
+
+  def onPageLoad(ern: String, searchOptions: GetDraftMovementsSearchOptions): Action[AnyContent] = {
+    authorisedDataRequestAsync(ern, searchMovementsBetaGuard(ern)) { implicit request =>
```

Comment:
Would this need to have the `searchMovementsBetaGuard(ern)` what is this used for?

Reply:
Also, the existing `DraftController` should be deleted.